flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/filesystem/FileSystemOutputFormat.java

```diff
@@ -194,6 +199,11 @@ public void close() throws IOException {
 			return this;
 		}
 
+		public Builder<T> setExtension(String extension) {
```

Comment:
`extension` is not easy to understand, can you use `OutputFileConfig` to support?